cli/dcoscli/package/main.py

```diff
@@ -517,3 +525,205 @@ def _uninstall(package_name, remove_all, app_id, cli, app):
         return 1
 
     return 0
+
+
+def _bundle(package_directory, output_directory):
+    """
+    :param package_directory: directory containing the package
+    :type package_directory: str
+    :param output_directory: directory where to save the package zip file
+    :type output_directory: str
+    :returns: process status
+    :rtype: int
+    """
+
+    if output_directory is None:
+        output_directory = os.getcwd()
+    logger.debug('Using [%s] as the ouput directory', output_directory)
+
+    # Find package.json file and parse it
+    package_json = _validate_json_file(
+        os.path.join(package_directory, 'package.json'))
+
+    with tempfile.NamedTemporaryFile() as temp_file:
+        with zipfile.ZipFile(
+                temp_file.name,
+                mode='w',
+                compression=zipfile.ZIP_DEFLATED,
+                allowZip64=True) as zip_file:
+            # list through package directory and add files zip archive
+            for filename in sorted(os.listdir(package_directory)):
```

Comment:
I was trying to keep the sha256 of the zip file constant across bundle operation. I found out it is not possible because of other things.